### ISSUE TITLE
Répliques à ajouter

### ISSUE THREAD

**Issue by Aluriak** [2ec0b4/kaamelott-soundboard#106]
Liste de répliques à incorporer (quand j'aurais le temps) : 

- [x] S01E02 : «Oui… Ou une fissure à colmater dans un muret.»
- [x] S01E12 : «Ah bah nous au bout d'un moment on a préféré plus rien dire.»
- [x] S01E12 : «J'ai rien compris.»
- [ ] S01E97 : «Kaamelott c'est pas une coopérative bovine, on a pas de vaches»
- [x] S02E05 : «C'est intéressant.»
- [ ] S02E05 : «J'ai déjà bossé avec des marteau, mais celle-là quand même on me l'avait jamais faite.»
- [x] S02E10 : «vous n'avez pas l'impression que je suis dans une baignoire ?»
- [ ] S02E33 : «C'est une blague ?»
- [ ] S02E33 : «C'est une conspiration !»
- [ ] S02E45 : «ah merde, j'avais oublié ça…»
- [ ] S02E61 : «Si c'est ça encore, ça va»
- [ ] S02E61 : «Cramez tout ça, je vous dis»
- [ ] S02E61 : «C'est pas les idées qui vous manque : c'est la conviction de devoir les réaliser»
- [ ] S02E71 : «Non, s'il vous plaît : fermez là, ou vous prenez une avoine.»
- [ ] S02E71 : «J'estime ne pas avoir à subir les fantasmes carriéristes d'une entité générationnelle, réactionnaire et oppressive.»
- [ ] S02E71 : «'Tain mais c'est pas con, ça !»
- [ ] S02E71 : «Mais évidemment, les gens sont pas débiles, ils voient bien que c'est du flan !»
- [ ] S02E73 : «attendez, je prend du singe pour la route !»
- [x] S02E86 : «Bah oui, c'est une rime triple ! Blanche et sèche… poitrine et prairIE de notre enfance»
- [ ] S03E39 : «Nan mais à côté de lui je fais noble, j'vous promet»
- [ ] S03E91 : «Mais lui on comprend ni c'qu'il dit, ni c'qu'il fait… C'est rare !», «Soutenez pas, vous surtout», «de toute façon, je vais vous dire, la nature elle est trop hostile pour un gars comme ça», «trop aimable», «est-ce que vous préparez un sale coups ?»
- [x] S04E03 : «Mais que se passe-t-il, j'entends crier ?»
- [x] S04E03 : «Je veux m'habiller de lierre, me coiffer de roseaux !»
- [x] S04E03 : «Vous allez me lâcher les noyaux, oui ?»
- [x] S04E03 : «Je crois qu'il faut se diriger petit à petit vers plus d'autonomie»
- [x] S04E03 : «Excusez-moi hein, je ne connais pas encore bien vos noms !»
- [x] S04E09 : «passez moi la… canne à pêche»
- [x] S04E48 : «mais moi je vous préviens, j'y connais rien en champignon»
- [x] S04E48 : «Bon ça va, si vous devez sonner l'alerte vous pouvez bien attendre que je revienne, non ?»
- [x] S04E49 : «Ouais, c'est mortel, ouais.»
- [ ] S04E57 : «Mais je le dis depuis une heure, moi ! Ça vaut pas le coups de ranger !»
- [ ] S04E57 : «Hé, quand je rentre, ça vous emmerde de vous occuper de moi, ou bien ?»
- [ ] S04E57 : «On vous a demandé le chemin de la plage, vous ?»
- [ ] S04E59 : «Vous êtes sûr que vous êtes pas un espion ?»
- [ ] S04E65 : «Ah non non, moi je parlais de leur jeter de la caillasse»
- [ ] S04E65 : «5 minutes… C'est une façon de parler, faut peut-être pas exagérer»
- [ ] S04E68 : «pourquoi vous me demandez alors ?»
- [ ] S04E68 : «Je vous ai dis que je faisais des potions avec, bon dieu ! Vous êtes bouché à la cire ou quoi ?»
- [x] S04E70 : «Je vais devenir… paladin !»
- [x] S04E70 : «Sire, vous me flattez»
- [x] S04E70 : «dans trois jours, c'est les vacances»
- [x] S04E78 : «C'est pas parce qu'ils ont trahis que ce sont plus des alliés.»
- [x] S04E78 : «Ah ouais ! Vous seriez une sorte de bi-taupe en fait !»
- [x] S04E78 : «simple déduction mon oncle !»

**Comment by 2ec0b4** [comment]
👌

C'est marrant, je pouvais modifier l'ordre de la liste juste avec un glissé-déposé. J'espère que tu n'as pas reçu une notification à chaque déplacement lol

**Comment by Aluriak** [comment]
Nope, et tu peux aussi cocher les items :) (là ça fera une notif, par contre)

Avec l'ordonancement, je m'aperçois qu'il y a un biais dans la représentation des livres… Est-ce le quatrième qui est particulièrement riche en répliques, ou juste ma lecture aléatoire qui est biaisée ?

ça donne des idées de feature à faire…

**Comment by Tagadda** [comment]
S01E07 Guenièvre "Allez-y, roulez roulez"
